cypress/integration/pages/storyPage/testsForCanonicalOnly.js

```diff
@@ -0,0 +1,27 @@
+// For testing important features that differ between services, e.g. Timestamps.
+// We recommend using inline conditional logic to limit tests to services which differ.
+export const testsThatAlwaysRunForCanonicalOnly = ({ service, pageType }) => {
+  describe(`No testsToAlwaysRunForCanonicalOnly to run for ${service} ${pageType}`, () => {});
+};
+
+// For testing feastures that may differ across services but share a common logic e.g. translated strings.
+export const testsThatFollowSmokeTestConfigForCanonicalOnly = ({
+  service,
+  pageType,
+}) =>
+  describe(`testsThatFollowSmokeTestConfigForCanonicalOnly for ${service} ${pageType}`, () => {
```

Comment:
This test is a duplicate of https://github.com/bbc/simorgh/blob/f53a329e6f227c379d868886b76745d06a116686/cypress/integration/pages/photoGalleryPage/testsForCanonicalOnly.js#L13. Which means they can be combined in https://github.com/bbc/simorgh/blob/latest/cypress/integration/pages/testsForAllCanonicalPages.js with an if statement for the two page types.